dandiapi/api/tests/test_dandiset.py

```diff
@@ -231,10 +231,7 @@ def test_dandiset_rest_create(api_client, user):
     assert dandiset.draft_version.metadata == {
         **metadata,
         'manifestLocation': [
-            (
-                f'https://api.dandiarchive.org/api/dandisets/{dandiset.identifier}'
-                f'/versions/draft/assets/'
-            )
+            f'http://localhost:9000/test-dandiapi-dandisets/test-prefix/dandisets/{dandiset.identifier}/draft/assets.yaml'  # noqa: E501
```

Comment:
Thank you. I find the breaking of long strings just to adhere to E501 to be something of a serious antipattern.